src/create_db.py

```diff
@@ -14,11 +14,174 @@
 
 def create_db():
     # open connection to database
-    print("Creating JobCompletion DB...")
-    conn = sqlite3.connect("JobCompletion.db")
-    print("Created JobCompletion DB sucessfully")
+    print("Creating Flux Accounting DB...")
+    conn = sqlite3.connect("FluxAccounting.db")
+    print("Created Flux Accounting DB sucessfully")
+
+    # User Table
+    print("Creating user_table in DB...")
+    conn.execute(
+        """
+            CREATE TABLE IF NOT EXISTS user_table (
+                creation_time bigint(20)            NOT NULL,
+                mod_time      bigint(20)  DEFAULT 0 NOT NULL,
+                deleted       tinyint(4)  DEFAULT 0 NOT NULL,
+                user_name     tinytext                        PRIMARY KEY,
+                admin_level   smallint(6) DEFAULT 1 NOT NULL
+        );"""
+    )
+    print("Created user_table successfully")
+
+    # Association Table
+    print("Creating association_table in DB...")
+    conn.execute(
+        """
+            CREATE TABLE IF NOT EXISTS association_table (
+                creation_time bigint(20)            NOT NULL,
+                mod_time      bigint(20)  DEFAULT 0 NOT NULL,
+                deleted       tinyint(4)  DEFAULT 0 NOT NULL,
+                id_assoc      integer                         PRIMARY KEY AUTOINCREMENT,
+                user          tinytext              NOT NULL,
```

Comment:
i notice "user" in this table, but "user_name" in the "user_table".  Just copied from slurm?

Reply:
Yes, the names of these fields were just copied from Slurm. We could (and probably should) rename these to keep them consistent and easy to interpret, especially because there are (at least three that I can think of) multiple ways to identify a user:

- Unix user ID
- User's "Association" ID
- User name (string)